generate-spec

```diff
@@ -249,6 +251,72 @@ if (file_exists($controllersDir)) {
 }
 
 $routes = [];
+foreach ($controllers as $controllerName => $stmts) {
+	$controllerClass = null;
+	/** @var Class_ $class */
+	foreach ($nodeFinder->findInstanceOf($stmts, Class_::class) as $class) {
+		if ($class->name->name === $controllerName . "Controller") {
```

Comment:
```suggestion
		if ($class->name->name === $controllerName . 'Controller') {
```

btw in PHP we use single quotes whenever possible, as double quotes would render/execute variables in case one slips in.